kahuna/package.json

```diff
@@ -49,6 +50,16 @@
             ]
           }
         }
+      },
+      "github:quantumui/titip@397fd324d5b5cc98ee332f876a100a70f69b273e": {
```

Comment:
It does, I guess we can just go for that. Worse case we can fork their repo and create a tag release if they can't pull their fingers out to push a git tag themselves.